### DISCUSSION TITLE
Stenographer live packet capture not showing in PCAP tab

### DISCUSSION THREAD

**Discussion by csev1755** [Security-Onion-Solutions/securityonion#5177] [Unsupported Versions] [answered]
Hopefully this is just a basic configuration error since I'm still very new to this.

I'm expecting to see data in the PCAP tab like in the documentation here:
[https://docs.securityonion.net/en/2.3/stenographer.html#stenographer](url)

But I don't see anything.

![image](https://user-images.githubusercontent.com/49533775/129771008-2200fd85-a78b-44f3-ae33-bc04d1e8912b.png)

I can see I have plenty of files within /nsm/pcap/ so I'm assuming I'm capturing packets, and the documentation makes it seem like they should just appear automatically. Why are they not showing?

Thanks in advance for any help.

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/pcap.html:
> You can pivot to PCAP from Alerts, Hunt, and Kibana. Alternatively, you can go directly to the PCAP interface and then put in your search criteria to search for a particular stream.

In most cases, you will be pivoting to the PCAP page from a specific alert or log in Alerts, Hunt, or Kibana as described at the following links:

https://docs.securityonion.net/en/2.3/alerts.html#actions

https://docs.securityonion.net/en/2.3/hunt.html#actions

https://docs.securityonion.net/en/2.3/kibana.html#transcript

**Reply by dylangerdaly** [reply]
The 'PCAP' action is missing from the 'Alerts' view, however it's there for Hunt.

**Reply by dougburks** [reply]
@dylangerdaly From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.

**Comment by dayton3375** [comment]
I have had this same issue, but found something interesting. Mine is recording pcaps of traffic destined to multicast or broadcast addresses such as ARP or Cisco Discovery Protocol. However, Zeek is recording the metadata of all traffic so I know Security Onion is seeing all the packets but Stenographer is only recording multicast/broadcast traffic. Is this the case for you?

I posted my question here: https://github.com/Security-Onion-Solutions/securityonion/discussions/5853

**Reply by dougburks** [reply]
@dayton3375 From #1720:

> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.
